src/vaibhav/exam1/ValidateUsername.java

```diff
@@ -0,0 +1,56 @@
+package vaibhav.exam1;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import vaibhav.base.Driver;
+
+public class ValidateUsername {
+	static WebDriver driver;
+	
+	public static void verifyUsername() {
+		boolean flag = true;
+		List<String>shortNameList = new ArrayList<String>();
+		List<String>userNameList = new ArrayList<String>();
+		driver.findElement(By.linkText("Demo Tables")).click();
+		int tableRows = driver.findElements(By.xpath("//table[@id='table1']/tbody/tr")).size();
+		for (int i = 1; i<=tableRows; i++) {
+			WebElement elementname = driver.findElement(By.xpath("//table[@id='table1']/tbody/tr["+i+"]/td[2]"));
+			String name = elementname.getText().toLowerCase();
+			String shortName =""+name.charAt(0);
+			System.out.println(shortName);
+			WebElement elementLastName = driver.findElement(By.xpath("//table[@id='table1']/tbody/tr["+i+"]/td[3]"));
+			String lastName = elementLastName.getText().toLowerCase();
+			System.out.println(lastName);
+			shortName += lastName;
+			shortNameList.add(shortName);
+			WebElement elementUserName = driver.findElement(By.xpath("//table[@id='table1']/tbody/tr["+i+"]/td[4]"));
+			String userName = elementUserName.getText();
+			userNameList.add(userName);
+		}
+		System.out.println(shortNameList);
+		System.out.println(userNameList);
+		for (int i = 0 ;i < shortNameList.size();i++) {
```

Comment:
You can compare data with single loop,

![image](https://user-images.githubusercontent.com/58640315/105631251-917bed00-5e73-11eb-9ffa-79beaf18e9c8.png)